src/Aquifer.Public.API/Endpoints/Bibles/Alignments/Greek/Endpoint.cs

```diff
@@ -0,0 +1,228 @@
+﻿using Aquifer.Common.Utilities;
```

Comment:
Much of this is copy/pasted from https://github.com/BiblioNexusStudio/aquifer-server/blob/master/src/Aquifer.API/Endpoints/Bibles/Alignments/Greek/Book/Chapter/Endpoint.cs as well as the `Request`, `Response`, and `Validator`.  I did modify it to expand out to full chapters or down to only words, though.